models/issue.go

```diff
@@ -1194,7 +1194,8 @@ func GetIssuesByIDs(issueIDs []int64) ([]*Issue, error) {
 // IssuesOptions represents options of an issue.
 type IssuesOptions struct {
 	db.ListOptions
-	RepoIDs            []int64 // include all repos if empty
+	RepoID             int64 // overwrites RepoCond if not 0
```

Comment:
the reason I added a RepoID instead of using RepoCond is, that I dont have to introduce builder as dependency into all places AND to write the same code on different places